library/src/test/kotlin/it/krzeminski/githubactions/IntegrationTest.kt

```diff
@@ -704,11 +709,39 @@ class IntegrationTest : FunSpec({
             on = listOf(Push()),
             sourceFile = Path.of("ExprIntegrationTest.kt"),
         ) {
+            val GREETING by Env
+            val FIRST_NAME by Env
+            val SECRET by Env
+            val TOKEN by Env
+            val SUPER_SECRET by Secrets
```

Comment:
Could we somehow differentiate how custom values are defined? Now it's not clear and one may think that `SUPER_SECRET` is a part of GitHub-provided context. How about:

```suggestion
            val SUPER_SECRET by Secrets.custom
```
? Any thoughts?

Reply:
Ok I wasn't aware of this naming convention :) Here I pointed it out looking purely on library's API. Let's leave it as is for now and see how it works in practice.